src/main/java/com/minecolonies/api/events/ColonyEvents.java

```diff
@@ -0,0 +1,38 @@
+package com.minecolonies.api.events;
+
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.colony.event.ColonyDeletedEvent;
+import com.minecolonies.api.util.Log;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.Event;
+
+/**
+ * Event manager for all forge events.
+ */
+public class ColonyEvents
+{
+    /**
+     * Event triggered when a colony is being deleted.
+     *
+     * @param colony the colony in question.
+     */
+    public static void deleteColony(final IColony colony)
```

Comment:
maybe we could make this sendSave a default method on the event? new ColonyDeletedEvent(colony).trigger();

Reply:
We'd need our own root event class in that case, the event it gets is just a forge event.

Reply:
Can just be an interface slapped on it.